webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -44,8 +55,23 @@ export class CourseWelcomeComponent implements OnInit {
   Users: any;
   role: string;
   selectedCourseId: string = '';
+  private API_URL = environment.API_URL;
 
   showInfoError: ShowInfoError;
+  isEditingName: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)